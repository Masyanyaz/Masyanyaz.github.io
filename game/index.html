<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Game</title>
	<link rel="stylesheet"  href="css/style.css">
</head>
<body>
	<canvas width="800" height="800" id="canvas"></canvas>

	<script src="js/script.js"></script>
</body>
</html>