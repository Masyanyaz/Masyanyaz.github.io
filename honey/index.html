<!DOCTYPE html>
<html lang="en">
<head>
    <title>Maksim Bryachnev</title>
    <meta charset="utf-8">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
    <link rel="stylesheet" href="css/bootstrap-grid.min.css">
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
<header class="header">
    <div class="header-container">
        <div class="row cap">
            <div class="col-auto cap-logo">
                <a href="index.html">
                    <img src="img/logo.png" alt="logo">
                </a>
            </div>
            <div class="col-auto">
                <a href="#stock" class="cap-sale">Акция!</a>
            </div>
            <div class="col-6">
                <ul class="cap-menu">
                    <li><a href="#product">Наша продукция<i class="fa fa-angle-down"></i></a></li>
                    <li><a href="#whyWe">О компании</a></li>
                    <li><a href="#delivery">Доставка</a></li>
                    <li><a href="#reviews">Отзывы</a></li>
                </ul>
            </div>
            <ul class="col-auto cap-contacts">
                <li>+375(00) 000-00-00</li>
                <li>+375(00) 000-00-00</li>
                <li class="cap-contacts-time">с 09:00 до 24:00 без выходных</li>
            </ul>
        </div>
    </div>
</header>
<div class="home" id="home">
    <div class="home-container">
        <div class="row">
            <div class="col-7 offset-5">
                <div class="card">
                    <p class="card-text">
                        Напрямую с пасеки, без наценок, <br> с быстрой доставкой по Минску
                    </p>
                    <div class="card-order">
                        <input type="text" class="card-order_name" placeholder="Ваше имя">
                        <input type="text" class="card-order_name" placeholder="Ваш телефон">
                        <a href="#" class="card-order_btn">Заказать консультацию</a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
<div class="home-down">
    <div class="home-container">
        <div class="row home-down-row">
            <div class="col-3">
                <div class="services">
                    <div class="services-item">
                        <img src="img/shield.png" alt="">
                        <p class="services-item_text">100 % <br>натуральный</p>
                    </div>
                </div>
            </div>
            <div class="col-2">
                <div class="services">
                    <div class="services-item">
                        <img src="img/wallet.png" alt="">
                        <p class="services-item_text">Лучшая цена</p>
                    </div>
                </div>
            </div>
            <div class="col-auto">
                <div class="services">
                    <div class="services-item">
                        <img src="img/delivery-truck.png" alt="">
                        <p class="services-item_text">Бесплатная <br>доставка от 40 руб.</p>
                    </div>
                </div>
            </div>
            <div class="col-3">
                <div class="services">
                    <div class="services-item">
                        <img src="img/growth.png" alt="">
                        <p class="services-item_text">Экологически <br>чистый</p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
<section class="stock" id="stock">
    <div class="container">
        <div class="row">
            <div class="col-xl-6 offset-6">
                <img src="img/stock.png" alt="stock">
            </div>
            <div class="col-xl-11">
                <p class="stock-text">
                    Только до <span>2019 г.</span> при покупке мёда на сумму от 50 бел. руб.,
                    получи скидку <span>10%</span> на следуюмщий заказ!
                </p>
                <p class="stock-text">В наличии свежий мёд 2018 года!</p>
            </div>

        </div>
    </div>
</section>
<section class="product" id="product">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Наша продукция
                    </h2>
                </div>
                <div class="toggle">
                    <a  class="toggle-btn toggle-active">
                        Цветочный мёд
                    </a>
                    <a  class="toggle-btn">
                        Подарочные наборы
                    </a>
                </div>
            </div>
        </div>
        <div class="row toggle-row active">
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-1.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-3.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-1.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-3.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-3.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-1.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-1.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med-3.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text">
                        <span class="product-item_text-span"><span>11.49</span> бел.руб.</span>
                        <span>10.49 бел.руб.</span>
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med3-1.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">14</span> бел.руб.
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med3-2.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">14</span> бел.руб.
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med3-3.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">14</span> бел.руб.
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/med3-4.png" alt="honey">
                    <h4 class="product-item_title">
                        Луговой
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">15</span> бел.руб.
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
        </div>
        <div class="toggle-row hidden">
            <div class="row-last">
                <div class="col-md-3">
                    <div class="product-item">
                        <img src="img/nab1.png" alt="honey" class="product-item_img">
                        <h4 class="product-item_title">
                            Подарочный мёд
                        </h4>
                        <p class="product-item_text small">
                            <span class="small-span">20</span> бел.руб.
                            за 1 кг
                        </p>
                        <a href="#" class="product-item_btn btn">
                            Узнать подробнее
                        </a>
                    </div>
                </div>
                <div class="col-md-3">
                    <div class="product-item">
                        <img src="img/nab2.png" alt="honey" class="product-item_img">
                        <h4 class="product-item_title">
                            Подарочный
                            мёд
                        </h4>
                        <p class="product-item_text small">
                            <span class="small-span">27</span> бел.руб.
                            за 1 кг
                        </p>
                        <a href="#" class="product-item_btn btn">
                            Узнать подробнее
                        </a>
                    </div>
                </div>
                <div class="col-md-3">
                    <div class="product-item">
                        <img src="img/nab3.png" alt="honey" class="product-item_img">
                        <h4 class="product-item_title">
                            Иван-чай
                        </h4>
                        <p class="product-item_text small">
                            <span class="small-span">10</span> бел.руб.
                            за 100 г
                        </p>
                        <a href="#" class="product-item_btn btn">
                            Узнать подробнее
                        </a>
                    </div>
                </div>
            </div>
            <div class="row-last">
                <div class="col-md-3">
                    <div class="product-item">
                        <img src="img/nab4.png" alt="honey" class="product-item_img">
                        <h4 class="product-item_title">
                            Оформление
                            подарочного мёда
                        </h4>
                        <p class="product-item_text small">
                            <span class="small-span">10</span> бел.руб.
                            за 50 г
                        </p>
                        <a href="#" class="product-item_btn btn">
                            Узнать подробнее
                        </a>
                    </div>
                </div>
                <div class="col-md-3">
                    <div class="product-item">
                        <img src="img/nab5.png" alt="honey" class="product-item_img">
                        <h4 class="product-item_title">
                            Бонбоньерки
                            на свадьбу
                        </h4>
                        <p class="product-item_text small">
                            <span class="small-span">2</span> бел.руб.
                            за 10 г
                        </p>
                        <a href="#" class="product-item_btn btn">
                            Узнать подробнее
                        </a>
                    </div>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <p class="ps">
                    * можно   выбрать  люой мёд и седелать его с прополисом
                </p>
                <div class="title">
                    <h2 class="title-text_product">
                        Продукты пчеловода
                    </h2>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/prod_pchel-1.png" alt="honey" class="product-item_img">
                    <h4 class="product-item_title">
                        Соты
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">20</span> бел.руб.
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/prod_pchel-2.png" alt="honey" class="product-item_img">
                    <h4 class="product-item_title">
                        Воск
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">27</span> бел.руб.
                        за 1 кг
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/prod_pchel-3.png" alt="honey" class="product-item_img">
                    <h4 class="product-item_title">
                        Перга
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">10</span> бел.руб.
                        за 100 г
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/prod_pchel-4.png" alt="honey" class="product-item_img">
                    <h4 class="product-item_title">
                        Пыльца
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">10</span> бел.руб.
                        за 100 г
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>

        </div>
        <div class="row-last">
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/prod_pchel-5.png" alt="honey" class="product-item_img">
                    <h4 class="product-item_title">
                        Прополис
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">10</span> бел.руб.
                        за 50 г
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
            <div class="col-md-3">
                <div class="product-item">
                    <img src="img/prod_pchel-6.png" alt="honey" class="product-item_img">
                    <h4 class="product-item_title">
                        Подмор
                    </h4>
                    <p class="product-item_text small">
                        <span class="small-span">2</span> бел.руб.
                        за 10 г
                    </p>
                    <a href="#" class="product-item_btn btn">
                        Узнать подробнее
                    </a>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="whyWe" id="whyWe">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title whyWe-title">
                    <h2 class="title-text whyWe-title-text">
                        Дарим подарки
                    </h2>
                </div>
                <div class="presents">
                    <img src="img/pod2.png" alt="">
                    <img src="img/pod1.png" alt="">
                    <p class="presents-text">
                        Каждому попупателю  при заказе от 40 бел.руб. <br>
                        вкусные соты в подарок!
                    </p>
                </div>
                <div class="title whyWe-second-title">
                    <h2 class="title-text whyWe-second-title-text">
                        Почему <br>
                        мёд покупают у нас
                    </h2>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <div class="objects">
                    <div class="objects-item">
                        <div class="objects-item-first">
                            <img src="img/01.png" alt="">
                            <p class="objects-text">
                                У нас всегда
                                100% натуральный мёд,
                                не разбавленный
                            </p>
                        </div>
                        <div class="objects-item-second">
                            <img src="img/02.png" alt="">
                            <p class="objects-text">
                                У нас лучшая цена,
                                так как мы являемся
                                производителями
                            </p>
                        </div>
                    </div>
                    <div class="objects-item">
                        <div class="objects-item-third">
                            <img src="img/03.png" alt="">
                            <p class="objects-text">
                                Всегда <br>
                                свежий мёд
                            </p>
                        </div>
                        <div class="objects-item-fourth">
                            <img src="img/04.png" alt="">
                            <p class="objects-text">
                                Не продаем
                                растопленный
                                прошлогодний мёд
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <div class="fact">
                    <p class="fact-title">Важный факт!</p>
                    <p>Внимание! Участился обман!</p>
                    <p class="fact-title-honey">В нашем мёде</p>
                    <div class="fact-pros">
                        <p>Нет патоки</p>
                        <p>Нет крахмала</p>
                        <p>Нет антибиотиков</p>
                        <p>Нет инвертного сахара</p>
                        <p>Нет воды</p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="cosmetic">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Мы используем мёд в косметике
                    </h2>
                </div>
            </div>
        </div>
        <div class="cosmetic-row">
            <div class="col-md-4 cosmetic-row-col">
                <div class="cosmetic-item">
                    <img src="img/kosm_01.png" alt="">
                    <p class="cosmetic-item-text">
                        Мёд для приготовления крема для рук
                    </p>
                </div>
            </div>
            <div class="col-md-4 cosmetic-row-col">
                <div class="cosmetic-item">
                    <img src="img/kosm_02.png" alt="">
                    <p class="cosmetic-item-text">
                        Мёд для приготовления маски для лица
                    </p>
                </div>
            </div>
            <div class="col-md-4 cosmetic-row-col">
                <div class="cosmetic-item">
                    <img src="img/kosm_03.png" alt="">
                    <p class="cosmetic-item-text">
                        Мёд для приготовления спа процедур
                    </p>
                </div>
            </div>
            <div class="col-md-4 cosmetic-row-col">
                <div class="cosmetic-item">
                    <img src="img/kosm_04.png" alt="">
                    <p class="cosmetic-item-text">
                        Мёд для ежедневного ухода за телом
                    </p>
                </div>
            </div>
            <div class="col-md-4 cosmetic-row-col">
                <div class="cosmetic-item">
                    <img src="img/kosm_05.png" alt="">
                    <p class="cosmetic-item-text">
                        Мёд при правильном питании
                    </p>
                </div>
            </div>
            <div class="col-md-4 cosmetic-row-col">
                <div class="cosmetic-item">
                    <img src="img/kosm_06.png" alt="">
                    <p class="cosmetic-item-text">
                        Мёд для приготовления блюд
                    </p>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="videoLife">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Видео с нашей пасеки
                    </h2>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <iframe class="videoLife-first" scrolling="no"
                        src="img/video.jpg" frameborder="0"></iframe>
            </div>
        </div>
    </div>
</section>
<section class="work">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Схема работы
                    </h2>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <img src="img/shema.png" alt="">
                <div class="work-step">
                    <p>Оформление заявки, связываемся с Вами</p>
                    <p>Формируем заказы, готовим к отправке</p>
                    <p>Доставка мёда к Вам домой</p>
                    <p>Получаете товар и наслаждаетесь мёдом</p>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="delivery" id="delivery">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Доставка
                    </h2>
                </div>
            </div>
            <div class="row">
                <div class="col-md-6">
                    <div class="delivery-truck">
                        <img src="img/dost.png" alt="">
                    </div>
                </div>
                <div class="col-md-5 offset-xl-1">
                    <div class="delivery-text">
                        <p class="delivery-text-item">
                            Доставим
                            по Минску быстро в день заказа
                        </p>
                        <p class="delivery-text-item">
                            Вы можете сделать
                            любые проверки мёда
                            в присутствии нашего курьера
                        </p>
                        <p class="delivery-text-item">
                            Есть какие-либо подозрения? <br>
                            Мёд заберем обратно
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="reviews" id="reviews">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Отзывы о нас
                    </h2>
                </div>
            </div>
        </div>
    </div>
</section>
<section class="question">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="title">
                    <h2 class="title-text">
                        Остались вопросы?
                    </h2>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-md-5 offset-xl-1">
                <div class="question-text">
                    <p class="question-text-item">
                        <span class="question-text-item_span">Время работы</span> <br>
                        Ежедневно с 9:00 до 24:00
                    </p>
                    <p class="question-text-item">
                        <span class="question-text-item_span">Время доставки</span> <br>
                        Ежедневно с 16:00 — 22:00
                    </p>
                    <p class="question-text-item">
                        <span class="question-text-item_span">При заказе до 15:00</span>
                        <br> доставка осуществляется <br> в тот же день
                    </p>
                    <p class="question-text-item">
                        <span class="question-text-item_span">Бесплатная доставка</span>
                        <br> при заказе на сумму от 40 руб.
                    </p>
                    <p class="question-text-item">
                        Доставка на сумму до 40 руб. — 5 руб.
                        При самовывозе уточняйте адрес
                    </p>
                </div>
            </div>
            <div class="col-md-6">
                <div class="question-form">
                    <p class="question-form-text">
                        Задайте вопрос <br>
                        нашим консультантам
                    </p>
                    <input type="text" class="question-form-input" placeholder="Ваше имя">
                    <input type="text" class="question-form-input" placeholder="Ваш телефон">
                    <input type="text" class="question-form-input" placeholder="Ваше сообщение">
                    <a href="#" class="question-form-btn btn">Получить</a>
                </div>
            </div>
        </div>
    </div>
</section>
<footer class="footer">
    <div class="container">
        <div class="footer-row">
            <div class="col-md-2">
                <div class="footer-ip">
                    <p>ИП</p>
                    <p>УНП <br> 220055, г. Минск</p>
                </div>
            </div>
            <div class="col-md-4">
                <h1 class="footer-title">Натуральный мёд</h1>
            </div>
            <div class="col-md-3">
                <div class="footer-contacts">
                    <p>
                        +375(00) 000-00-00 <br>
                        +375(00) 000-00-00 <br>
                        xxxxxxxx@yandex.by
                    </p>
                    <p class="footer-contacts_span">Разработка сайта: Bryachnev Maksim</p>
                </div>
            </div>
        </div>
    </div>
</footer>
<script src="js/jquery.js"></script>
<script src="js/script.js"></script>
</body>
</html>






















