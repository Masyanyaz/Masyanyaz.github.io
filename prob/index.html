<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8">
	<title>js</title>
	<link rel="stylesheet" href="style.css">
</head>
<body>
	


<script src="js.js"></script>
</body>
</html>