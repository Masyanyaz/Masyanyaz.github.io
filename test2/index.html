<!DOCTYPE html>
<html>
<head>
	<title>Мой сайт</title>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<style>
		ul, li, body, html {
			margin: 0;
		}

		img {
			padding-right: 22px;

		}

		.content{
			max-width: 660px;
		}

		.content h3 {
			font-size: 18px;
			line-height: 24px;
			color: #bf7800;
			font-weight: bold;
			margin-bottom: 8px;
		}

		.header {
			display: flex;
			padding: 7px 0 10px;
			flex-wrap: wrap;
		}

		.header::after {
			content: '';
			width: 100%;
			height: 2px;
			background: #c73728;
			display: block;
		}

		.information {
			display: flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
		}

		@media (max-width: 580px) {
			html {
				font-size: 14px;
			}

			.text {
				width: 225px;
			}

			img {
				padding-right: 5px;
				width: 35%;
				height: 100%;
				min-width: 155px;
			}
		}
		@media (max-width: 425px) {
			.information {
				justify-content: center;
			}
		}


	</style>
</head>
<body>
<div class="content">
	<div class="header">
		<h3 style="padding: 0 23px 0 36px;">День 1</h3>
		<h3 style="color: black; padding: 0 23px;">Прибытие</h3>
	</div>
	<div class="information">
		<div style="padding-bottom: 8px">
			<ul class="text">
				<li>Прибытие в Санкт-Петербург</li>
				<li>Трансфер в центр города</li>
				<li>Заселение в отель по вашему выбору:
					<ul>
						<li>отель 3*</li>
						<li>квартира</li>
						<li>комната у жителя</li>
					</ul>
				</li>
			</ul>
		</div>
		<img width="250" alt="Saint-Pétersbourg" src="http://lati.me/images/photos/medium/article442.jpg" height="166" title="Saint-Pétersbourg">
	</div>
</div>

</body>
</html>
