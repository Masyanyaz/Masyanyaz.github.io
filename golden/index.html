<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Servises</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles_first">
					Welcome To Our Studio!
				</div>
				<h1>
					It's nice to Meet you
				</h1>
			</div>
			<a href="#" class="button">Tell Me More</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="title">
				<h2>
					Services
				</h2>
				<p>
					Proin iaculis purus conseqquat sem core.
				</p>
			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon1.png" alt="Услуга">
					<h3>
						E-Commerce
					</h3>
					<p>
						shfkhasflkashfklashfklashfklas
					</p>
				</div>
				<div class="services_item">
					<img src="img/icon2.png" alt="Услуга">
					<h3>
						Responsive Web
					</h3>
					<p>
						shfkhasflkashfklashfklashfklas
					</p>
				</div>
				<div class="services_item">
					<img src="img/icon3.png" alt="Услуга">
					<h3>
						Web Security
					</h3>
					<p>
						shfkhasflkashfklashfklashfklas
					</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>
					Our portfolio
				</h2>
				<p>
					Proin iaculis purus conseqquat sem core.
				</p>
			</div>
			<div class="portfolio clearfix">
				<div class="works clearfix">
					<img src="img/img1.jpg" alt="Работа">
					<img src="img/img2.jpg" alt="Работа">
					<img src="img/img3.jpg" alt="Работа">
				</div>
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="title">
				<h2>
					About us
				</h2>
				<p>
					Proin iaculis purus conseqquat sem core.
				</p>
			</div>
			<div class="about clearfix">
				<div class="about_item">
					<img src="img/about1.jpg" alt="about">
					<div class="about_text">
						<h3>
							ghbdtn
						</h3>
						<p>
							asdasdasfsds<br>jfshdlfsdhflksdh<br>flksdhfklsdfhls<br>kdfhsdkfhsdklfhsdkgf<br>hskdlfhsdklf
						</p>
					</div>
				</div>
				<div class="about_itemr">
					<img src="img/about2.jpg" alt="about">
					<div class="about_textr">
						<h3>
							ghbdtn
						</h3>
						<p>
							asdasdas
						</p>
					</div>
				</div>
				<div class="about_item">
					<img src="img/about3.jpg" alt="about">
					<div class="about_text">
						<h3>
							ghbdtn
						</h3>
						<p>
							asdasdas
						</p>
					</div>
				</div>
				<div class="about_itemr">
					<img src="img/about4.jpg" alt="about">
					<div class="about_textr">
						<h3>
							ghbdtn
						</h3>
						<p>
							asdasdas
						</p>
					</div>
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<p>
				2014-2015
			</p>
		</div>
	</footer>
</body>
</html>